src/hookable.ts

```diff
@@ -149,8 +151,11 @@ export class Hookable <
         }
       })
     }
-    if (this._after && event) {
-      callEachWith(this._after, event)
+    if (event) {
+      if (this._after) {
+        callEachWith(this._after, event)
+      }
+      this._count--
```

Comment:
Might be over complexity but shall we split count by name using a map? this._activeHooks<name, number>

Reply:
Also is it possible we somehow move counter implementation to debugger only?